monitoring/api/v3/alerts-client/snippets_test.py

```diff
@@ -102,15 +103,23 @@ def test_enable_alert_policies(capsys, pochan):
     def invoke_sample(val):
         snippets.enable_alert_policies(pochan.project_name, val)
 
+    # These sleep calls are for mitigating the following error:
+    # "409 Too many concurrent edits to the project configuration.
+    # Please try again."
+    # TODO(#3310): Having multiple projects will void these `sleep()` calls
```

Comment:
nit: I prefer not putting TODO(issues) in code - IMO the issue should have a one way link to the code in question, but if we decide to close this issue for different reason, another commit is required to remove this TODO

Reply:
OK